packages/atomic/src/utils/local-storage-utils.tsx

```diff
@@ -0,0 +1,108 @@
+import {SearchEngine} from '@coveo/headless';
+
+export enum StorageItems {
+  RECENT_QUERIES = 'coveo-recent-queries',
```

Comment:
seems a bit out of place here 🤔  Why not keep this in the recent queries component and simply accept a string as a key ?

Reply:
Alternatively, if we choose to keep `coveo-recent-queries` inside the safe storage file, we could make `clear`, `getItem`, `removeItem`, and the rest `private` and we could introduce public setters & getters instead. E.g.:
```ts
this.storage = new SafeStorage(this.bindings.engine.logger);
...
this.storage.recentQueries = this.recentQueriesList.state.queries;
```

Reply:
I wanted to add types to set and get values, so that we can get some standardization on the format of keys (eg: always same prefix, always hyphen separated)